CI/gen_doc/docs/sysobjs.json

```diff
@@ -150,29 +150,34 @@
     "props": [
       {
         "prop_name": "EnabledChannels",
-        "prop_title": "  EnabledChannels Enabled Channels",
+        "prop_title": " EnabledChannels Enabled Channels",
         "prop_description": "Indexs of channels to be enabled. Input should be a [1xN] vector with the indexes of channels to be enabled. Order is irrelevant"
       },
       {
         "prop_name": "SampleAveragingLength",
-        "prop_title": "  SampleAveragingLength",
-        "prop_description": "Block length of samples to be averaged. Applied in the Averaging Mode register only when OUT_DATA_MD is set to 30-bit averaged differential modeHelp for adi.AD4630_24.Rx/SampleAveragingLength is inherited from superclass adi.AD463x.Base"
+        "prop_title": "SampleAveragingLength",
+        "prop_description": "Block length of samples to be averaged. Applied in the Averaging Mode register only when OUT_DATA_MD is set to 30-bit averaged differential mode"
       },
       {
         "prop_name": "SampleRate",
-        "prop_title": "  SampleRate Sample Rate",
-        "prop_description": "Baseband sampling rate in Hz, specified as a scalar in samples per second.Help for adi.AD4630_24.Rx/SampleRate is inherited from superclass adi.AD463x.Base"
+        "prop_title": "SampleRate - Sample Rate",
+        "prop_description": "Baseband sampling rate in Hz, specified as a scalar in samples per second."
       },
       {
         "prop_name": "SamplesPerFrame",
-        "prop_title": "  SamplesPerFrame Samples Per Frame",
-        "prop_description": "Number of samples per frame, specified as an even positive integer.Help for adi.AD4630_24.Rx/SamplesPerFrame is inherited from superclass adi.AD463x.Base"
+        "prop_title": "  Frame size",
+        "prop_description": "Size of the frame in samplesHelp for adi.AD4630_24.Rx/SamplesPerFrame is inherited from superclass matlabshared.libiio.base"
       },
       {
         "prop_name": "uri",
         "prop_title": "  URI - remote host URI",
         "prop_description": "Hostname or IP address of remote libIIO deviceHelp for adi.AD4630_24.Rx/uri is inherited from superclass matlabshared.libiio.base"
       }
     ]
+  },
+  {
+    "name": "adi.AD4020.Rx",
```

Comment:
Unlike ad2s120, properties didn't get filled out for AD4020. So documentation wouldn't get generated. 

Just to confirm did you run the gen_sysobj_doc.m with the most recent driver version checked out?

Reply:
I noticed the difference with ad2s1210 but can't figure out why properties are not filled out for ad4020 : I'm running gen_sysobj_doc.m with the same Matlab version, just checking out the relevant branch for each device in my git workspace.

Reply:
I tried to replicate the behavior locally. I cloned the repo, switched to this branch, and executed gen_sysobj_doc - I could see the properties appear correctly in sysobjs.json. 
![image](https://github.com/analogdevicesinc/PrecisionToolbox/assets/108716875/08daf8a9-4794-4f9d-a18b-671c66b79c54)

Reply:
Is there another clone (perhaps with the early version of the driver or something) on the MATLAB path, that's taking precedence? That could explain why the properties don't show up.

Reply:
Did the same on my side : cloned the repo in another directory and executed gen_sysobj_doc in that new directory with the same result and I still cannot see the properties in sysobjs.json for AD4020.

Reply:
@ribdp Would you please consider pushing your sysobjs.json changes with the properties updated since this appears to be a setup issue on my side ?
I can't figure out why the properties do not show after executing gen_sysobj_doc.m in this case, even looking at MATLAB path for a probable cause.

Reply:
@apelete Sure thing. I've pushed the changes now